src/javascript/EditPanel/EditPanelContent/EditPanelContent.jsx

```diff
@@ -34,17 +34,11 @@ const styles = theme => ({
 
 export const EditPanelContent = ({classes, isDirty}) => {
     const {mode} = useContentEditorConfigContext();
+    const {hasPreview} = useContentEditorContext();
+
     return (
         <>
-            {mode === 'create' ?
-                (
-                    <FullWidthContent
-                        classes={{root: classes.fullWidthRoot}}
-                        data-sel-mode={mode}
-                    >
-                        <FormBuilder mode={mode}/>
-                    </FullWidthContent>
-                ) :
+            {hasPreview ?
                 (
                     <>
                         <PublicationInfoProgress/>
```

Comment:
So publicationInfoProgress is only available when you have a preview now ?
doesnt seem's right :)

Reply:
thanks !